src/index.test.js

```diff
@@ -36,6 +36,10 @@ function renderComponent(props: Object = {}) {
   )
 }
 
+function expectedHeaders(headers) {
+  return
+}
+
```

Comment:
Unused?

Reply:
indeed. that was from the DRY version of the fetch tests, must have missed it when switching back to inline.